docs/conf.py

```diff
@@ -112,10 +112,8 @@
     "source_branch": "main",
     "source_directory": "docs/",
     "light_css_variables": {
-        "color-brand-primary": "#1C7362",
-        "color-brand-content": "#1C7362",
-    },
```

Comment:
These weren't used since `light_css_variables` had two entries in the dictionary definition.